src/setupTests.ts

```diff
@@ -1,32 +1,39 @@
 import "isomorphic-fetch";
 import { jest } from "@jest/globals";
+import { MockInstance } from "jest-mock";
 import { OrgSettings } from "./getOrgSettings";
+import { merge } from "lodash-es";
+
+let mocks = {};
 
 beforeEach(() => {
+  mocks = {};
+
   const mainKv: MockCloudflareKV = {
     get: jest.fn(),
-    mockKv(mocks: KvMocks) {
-      const kv: jest.Mock<any> = global.MAIN_KV.get as jest.Mock<any>;
-      kv.mockImplementation(async (key: string) => {
```

Comment:
Calling `mockImplementation` inside of the `mockKv()` function was a mistake because it meant that the mock didn't work for tests that don't call `mockKv()`. Some tests just want default functionality and they shouldn't have to call `mockKv()` to get that

Reply:
This makes sense.  Thank you for explaining this change.